Dockerfile

```diff
@@ -24,12 +22,10 @@ COPY ./Gemfile.lock ./Gemfile /opt/app-root/src/
 
 RUN FULL_RHEL=$(microdnf repolist --enabled | grep rhel-8);                                   \
     if [ -z "$FULL_RHEL" ] ; then                                                             \
-      rpm -Uvh $pgRepo $pgRepoKey                                                          && \
-      sed -i 's/^\(enabled.*\)/\1\nincludepkgs=postgresql\*/;' /etc/yum.repos.d/CentOS*.repo; \
```

Comment:
This is being removed. I highly recommend to explicitly set packages that are enabled.